app/src/scenes/Home/index.tsx

```diff
@@ -16,13 +16,23 @@ class Home extends React.Component {
         <Col md={24} lg={21}>
           <Row className={styles.mainContent}>
             <Col md={10} className={styles.descriptionContent}>
-              <img src={logo} alt="Academix Logo" width={350} />
+              <img src={logo} alt="Academix Logo" className={styles.logo} />
               <p className={styles.description}>
                 AcadeMix aims to curate a collection of free education resources
                 available for Sri Lankan students to maximise their learning
                 potential from Primary through to Post Graduate education and
                 beyond.
               </p>
+              <p className={styles.descriptionContact}>
+                If you have a website, blog or channel that features free
+                education content to support curriculum in Sri Lanka, do share
+                with us at{' '}
+                <a href="https://mail.google.com/mail/u/0/?view=cm&fs=1&to=info@sefglobal.org&tf=1">
```

Comment:
This should be mailto link

Reply:
https://www.w3schools.com/tags/tag_address.asp

Reply:
Done! Thanks @jayasanka-sack